trin-storage/src/versioned/sql.rs

```diff
@@ -0,0 +1,14 @@
+pub const STORE_INFO_CREATE_TABLE: &str = "
+    CREATE TABLE IF NOT EXISTS store_info (
+        content_type TEXT PRIMARY KEY,
+        version TEXT NOT NULL
+    )";
```

Comment:
Is it worth adding a timestamp here, so that we can identify when the db was last updated?

Reply:
I don't have a great reason... Maybe just to check when the db was last migrated? But I'm not sure that's a strong reason, since the version to which it was last migrated is already available, and that's the important information. So feel free to disregard